pkg/awsapi/iam.go

```diff
@@ -405,27 +419,36 @@ func (i *IAM) AttachInlineRolePolicy(ctx context.Context, req IAMRoleRequest) (*
 	return &IAMRoleResponse{}, nil
 }
 
-//GetRole gets the role from aws iam
-func (i *IAM) GetRole(ctx context.Context, req IAMRoleRequest) (*iam.GetRoleOutput, error) {
-	log := log.Logger(ctx, "awsapi", "iam", "GetRole")
+// CreateRole will try to create an IAM Role, or return back Nil if it does not exist
```

Comment:
`or return back Nil if it can not be created`

Reply:
:+1: